codegen/render/funcs.go

```diff
@@ -562,6 +571,341 @@ func fromJSON(str string) map[string]interface{} {
 	return m
 }
 
+// toJSONSchema generates the json schema for the values of the helm chart
+func toJSONSchema(values values.UserHelmValues) string {
+	reflector := createJsonSchemaReflector(values)
+	schema := new(jsonschema.Schema)
+	schema.Version = "https://json-schema.org/draft/2020-12/schema"
```

Comment:
Where did this come from? Maybe a comment or link?

Reply:
fixed, I got this from the jsonschema package.